src/com/lilithsthrone/game/character/persona/Name.java

```diff
@@ -724,6 +796,10 @@ private static NameTriplet getDemonName() {
 	}
 	
 	public static NameTriplet getRandomProstituteTriplet() {
-		return Util.randomItemFrom(prostitute);
+		// occasionally throw some "regular" names in there - 25% of the time
+		if(Util.random.nextInt(3)<3)
```

Comment:
This will always return true, as `Util.random.nextInt(3)` returns a number in the range of 0-2, inclusive.

As in the other comment I made higher up, I would find it clearer if it were changed to:
`if(Math.random()<0.75)`